### PR TITLE
1158-config-insights-type-severity-filter-dropdown-items-incorrect

### LINKED ISSUE #1158: Config Insights Type and Severity Filter Dropdown Items are incorrect
image

image

### PR DESCRIPTION
**Pull request by mainawycliffe**
Closes #1158

### CODE REVIEW

File: src/components/ConfigAnalysis/ConfigAnalysisSeverityDropdown.tsx [hunk 1]
```diff
@@ -3,18 +3,42 @@ import { Control } from "react-hook-form";
...
+  HiInformationCircle,
+  HiOutlineChevronDoubleDown,
+  HiOutlineChevronDown,
+  HiOutlineMinus
 } from "react-icons/hi";
 import { defaultSelections } from "../Incidents/data";
 import { ReactSelectDropdown } from "../ReactSelectDropdown";
 
 export const configAnalysisSeverityItems = {
+  Low: {
```
Comment (moshloop) [inline src/components/ConfigAnalysis/ConfigAnalysisSeverityDropdown.tsx:15]: Can we not reuse the same constants as the incident severities ?

Comment (mainawycliffe) [reply src/components/ConfigAnalysis/ConfigAnalysisSeverityDropdown.tsx:15]: In config insights, we seem to have an extra severity - info, that doesn't exist in incidents severity, other than that, we can.

**Comment by moshloop** [comment]
@mainawycliffe 
- The info, critical, warning icons do not seem to be consistent, 
- The dropdowns should be sorted
- Can we change the font-size of all dropdown to text-sm ?

**Comment by moshloop** [comment]
@mainawycliffe  I don't see the size taking effect - I think react select is adding its own size

**Pull request merged**